fe1-web/src/features/rollCall/network/RollCallHandler.ts

```diff
@@ -158,28 +158,38 @@ export const handleRollCallCloseMessage =
         const hasToken = rollCall.containsToken(token);
         aDispatch(setLaoLastRollCall(laoId, rollCall.id, hasToken));
 
-        // If we had a token in this roll call, we subscribe to our own social media channel
-        if (token && hasToken) {
-          await subscribeToChannel(
-            laoId,
-            dispatch,
-            getUserSocialChannel(laoId, token.publicKey),
-          ).catch((err) => {
-            console.error(
-              `Could not subscribe to our own social channel ${token.publicKey}, error:`,
-              err,
-            );
-          });
-        }
+        const promises: Promise<unknown>[] = [];
+
+        // subscribe to all roll call attendee's social media channels
+        promises.push(
```

Comment:
I know this thing is very smart, but pretty please, split it into multiple variables because it's too nested to be intelligible without great effort 😁

Reply:
Slightly better indeed 😁